js/app.js

```diff
@@ -194,6 +248,9 @@
    *     event, 'light-green'
    */
 
+   btn13.addEventListener("click", function(event){
```

Comment:
The instructions asked to call bind with just one argument. Please update the bind function to only take one argument and update the following invocation to accept two arguments.